### ISSUE TITLE
The player settings button gets read twice by screen readers

### ISSUE THREAD

**Issue by pipoprods** [Chocobozzz/PeerTube#6780]
### Describe the current behavior

When using a screen reader we get two descriptions of the player setting button.

### Steps to reproduce

1. open a video URL
2. navigate in UI with a screen reader

### Describe the expected behavior

There should be only one description

### Additional information

(bug fixed, PR to come)

**Comment by Chocobozzz** [comment]
Thanks! This should be fixed by https://github.com/Chocobozzz/PeerTube/commit/93b353addecb6b2d449d432fa26b8d7a4593b43d
